packages/babel-types/scripts/generators/docs.mjs

```diff
@@ -1,10 +1,10 @@
 "use strict";
```

Comment:
There are a bunch of files that are now modules, and still have the `"use strict"` directive: that can be removed.